src/server.h

```diff
@@ -1401,6 +1401,7 @@ typedef struct {
     list        *history_aof_list;    /* HISTORY AOF list. When the AOFRW success, The aofInfo contained in
                                          `base_aof_info` and `incr_aof_list` will be moved to this list. We
                                          will delete these AOF files when AOFRW finish. */
+    aofInfo     *tmp_incr_aof_info;   /* Temp INCR file info(Used to accumulate data in AOF_WAIT_REWRITE state). */
```

Comment:
i was thinking that we should just put in the server struct, and that there's no need to store it in the manifest.
i see some advantages in storing it (like being able to clean it up after crash), but i'm not sure that's mandatory.
WDYT?

Reply:
i'm not sure it's the internal behavior of the manifest, and specifically, i don't think it should be persisted into the manifest **file**.
i think that temp file is part of the server state (waiting for the first rewrite to complete).
it's pretense in the manifest is somewhat undesired (if an external observer looks at the manifest file, ideally this temp file should not be listed IMHO).

regarding the file name, i think we could consider using the child process PID or anything else, and then rename it when the AOFRW is complete.
since the flush of the manifest is what matters, we anyway have an additional rename, so one more isn't a problem.

i don't insist on this change, but i have a feeling it has the potential to simply things greatly.

Reply:
I think this makes things more complicated. First, this will be a separate naming rule system (detached from the existing manifest mechanism and cannot reuse existing code), second, we are opening the INCR AOF file before fork (this is something we have been emphasizing before), So we can't assemble our INCR AOF filename like we named BASE AOF

Reply:
This way we will have two rename operations in doneHandler

Reply:
yes, that's what i was aiming for (a rename).. 
you're right that we can't use the PID, we wanted to be sure the disk is writable before forking.

Reply:
Code changed.